domains/beno.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Beno's personal developer website",
+  "repo": "https://github.com/benosambinu/benosambinu",
```

Comment:
Seems like you have not created the github pages repo yet. You can take a look at the [github pages guide](https://guides.github.com/features/pages/) if you have any trouble with that.